### PR TITLE
Clarify limitations of Azure File storage

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1601757
- https://github.com/openshift/openshift-docs/pull/14032

### CODE REVIEW

**Comment by liangxia** [comment]
LGTM (from QE)

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 1]
```diff
@@ -51,13 +50,36 @@ $ mount
 If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
 default value `0755` to `0777` or `0775`. This manual step is required because
 the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
-in {product-title} 3.9. The following examples show configuration files with the
-changed values.
+in {product-title} {product-version}. The following examples show configuration
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: Revert this change. The version listed is relevant to the statement.

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 2]
```diff
@@ -51,13 +50,36 @@ $ mount
...
+directory is different from the process UID of the container.
+
+[CAUTION]
+====
+You might experience instability in your environment if you use any container
+images that use unsupported file system features. Containers for PostgreSQL and
+MySQL are known to have issues when used with Azure File.
+====
+
+If you use MySQL containers, you must modify the PV configuration as a
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: I think that this process should have its own heading.

**Review by kalexand-rh — COMMENTED** [review]
I have two requests, but it largely LGTM.

File: install_config/persistent_storage/persistent_storage_azure_file.adoc (hunk 1 above)
Comment (jboxman) [reply install_config/persistent_storage/persistent_storage_azure_file.adoc]: Oh, oops. Thanks!
